### PR TITLE
OSRA 334 - sponsor#show views, actions

### PR DESCRIPTION
**Pull request by razorcd**
**OSRA 334 - Create #show views, actions**
https://osraav.atlassian.net/browse/OSRA-334

**"Currently Sponsored Orphan" and "Previously Sponsored Orphans" tables from the bottom of the sponsor#show view will be added in a different task.**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -9,4 +9,13 @@ def format_full_date(date)
   def format_month_year_date(date)
     (date || NullDate.new).strftime(MONTH_YEAR_DATE_FORMAT_STRING)
   end
+
+  def en_ar_country(country)
```
Comment (PurityControl) [inline app/helpers/application_helper.rb]: we already have this functionality merge in #291. Please can we remove it.

**Comment by PurityControl** [comment]
do we get a screenshot?

**Comment by razorcd** [comment]
sponosr show

**Comment by razorcd** [comment]
PR Updated

 - `Requested orphan count` is truncated in the view. That will be fixed with css. (http://www.w3schools.com/cssref/css3_pr_text-overflow.asp)

**Comment by PurityControl** [comment]
@razorcd can you make a jira story for the css and reference this pr in it please.

**Pull request merged**